packages/did-core-test-server/suites/did-consumption/did-json-consumption.js

```diff
@@ -0,0 +1,133 @@
+          'Array is added to the list in order, converted based on the JSON ' +
+          'representation type of the array value, as defined in this ' +
+          'table.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+        it('6.2.2 JSON Consumption - JSON Array where the data model entry ' +
+          'value is a set: A set, where each value of the JSON Array is ' +
+          'added to the set in order, converted based on the JSON ' +
+          'representation type of the array value, as defined in this ' +
+          'table.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else if(value !== null && typeof value === 'object') {
+        it('6.2.2 JSON Consumption - JSON Object: A map, where each member ' +
+          'of the JSON Object is added as an entry to the map. Each entry ' +
+          'key is set as the JSON Object member name. Each entry value is ' +
+          'set by converting the JSON Object member value according to the ' +
+          'JSON representation type as defined in this table. Since order ' +
+          'is not specified by JSON Objects, no insertion order is ' +
+          'guaranteed.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else if(typeof value === 'string' && isXmlDatetime(value)) {
+        it('6.2.2 JSON Consumption - JSON String where data model entry ' +
+          'value is a datetime: A datetime.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else if(typeof value === 'string') {
+        it('6.2.2 JSON Consumption - JSON String, where the data model ' +
+          'entry value type is string or unknown: A string.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else if(typeof value === 'number') {
+        if(!Number.isInteger(value)) {
+          it('6.2.2 JSON Consumption - JSON Number without a decimal or ' +
+            'fractional component: An integer.', () => {
+            expect(reserializationSuccess(value)).toBe(true);
+          });
+        } else {
+          it('6.2.2 JSON Consumption - JSON Number with a decimal and ' +
+            'fractional component, or when entry value is a double ' +
+            'regardless of inclusion of fractional component: ' +
+            'A double.', () => {
+            expect(reserializationSuccess(value)).toBe(true);
+          });
+        }
+      } else if(typeof value === 'boolean') {
+        it('6.2.2 JSON Consumption - JSON Boolean: A boolean.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else if(value === null) {
+        it('6.2.2 JSON Consumption - JSON null literal: A null value.', () => {
+          expect(reserializationSuccess(value)).toBe(true);
+        });
+      } else {
+        it('UNKNOWN JSON TYPE '+ value, () => {
+          expect(true).toBe(true);
```

Comment:
This should be an error.

Reply:
Fixed in 550fc1bd9b494843995c31c021047fcc54fd29a1.